support-frontend/assets/helpers/abTests/helpers.ts

```diff
@@ -1,130 +1,333 @@
 import type { Participations } from 'helpers/abTests/abtest';
-import type { ContributionAmounts } from 'helpers/contributions';
+import type {
+	AmountsTest,
+	SelectedAmountsVariant,
+} from 'helpers/contributions';
 import type { Settings } from 'helpers/globalsAndSwitches/settings';
+import type { IsoCountry } from 'helpers/internationalisation/country';
 import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
 
-type FallbackAmounts = Record<CountryGroupId, ContributionAmounts>;
-export const FALLBACK_AMOUNTS: FallbackAmounts = {
-	GBPCountries: {
-		ONE_OFF: {
-			amounts: [30, 60, 120, 240],
-			defaultAmount: 60,
-		},
-		MONTHLY: {
-			amounts: [3, 7, 12],
-			defaultAmount: 7,
-		},
-		ANNUAL: {
-			amounts: [60, 120, 240, 480],
-			defaultAmount: 120,
-		},
+export const FALLBACK_AMOUNTS: AmountsTest[] = [
+	{
+		testName: 'FALLBACK_AMOUNTS__GBPCountries',
+		liveTestName: '',
+		isLive: false,
```

Comment:
Is the purpose of these defaults so we fallback and render something if the amounts data from config isn't available? If so, should we be setting `isLive` to `false`? Won't this block them being rendered?

Reply:
We only have hardcoded default amounts for each region. And for region amounts test, setting the `isLive` flag to false tells the system to use the control variant for that test.

Reply:
Got it 👍